README.md

```diff
@@ -161,21 +171,29 @@ from aicsimageio import AICSImage
 
 # Get an AICSImage object
 img = AICSImage("my_file.tiff")  # selects the first scene found
-img.metadata  # returns the metadata object for this image type
+img.metadata  # returns the metadata object for this file format (XML, JSON, etc.)
 img.channel_names  # returns a list of string channel names found in the metadata
 img.physical_pixel_size.Z  # returns the Z dimension pixel size as found in the metadata
 img.physical_pixel_size.Y  # returns the Y dimension pixel size as found in the metadata
 img.physical_pixel_size.X  # returns the X dimension pixel size as found in the metadata
 ```
 
+### Format Conversion to OME-TIFF
+
+```python
+from aicsimageio import AICSImage
+
+AICSImage("my_file.czi").save("my_file.ome.tiff")
+```
```

Comment:
Add comment after this that details we aren't doing metadata conversion yet, just basic metadata creation.